graphql_compiler/schema_transformation/merge_schemas.py

```diff
@@ -0,0 +1,173 @@
+# Copyright 2019-present Kensho Technologies, LLC.
+from collections import namedtuple
+from copy import deepcopy
+
+from graphql import build_ast_schema
+from graphql.language import ast as ast_types
+from graphql.language.printer import print_ast
+import six
+
+from .utils import SchemaNameConflictError, check_ast_schema_is_valid, get_query_type_name
+
+
+MergedSchemaDescriptor = namedtuple(
+    'MergedSchemaDescriptor', (
+        'schema_ast',  # Document, AST representing the merged schema
+        'name_to_schema_id',  # Dict[str, str], type name to id of the schema the type is from
+    )
+)
+
+
+def _basic_schema_ast(query_type):
```

Comment:
Generally, it's good practice to name functions verb-like names (e.g. `_get_basic_schema_ast()`) instead of noun-like names (`_basic_schema_ast()`) -- this tells the reader what the function does when called.